woocommerce/woocommerce-ios
# Store creation: Make new profiler flow default

Part of: #10385 and #10374

## Description

Enables the optimized store profiler flow by default.

Changes
- Use `StoreProfilerAnswers` model to collect and store answers
- Present the free trail summary before the profiler flow
- Track `site_creation_step` events from the new container view

To be done in future PRs
- Upload the answers after user lands on the dashboard screen
- Remove the local notification after entering store name
- Remove `SiteProfilerData` and its usage

## Testing instructions

- Log in and switch to the Menu tab.
- Select the store picker and tap Add a store > Create a new store.
- You should see the Free trial summary screen.
- Tap the "Try for Free" button
- After the loading indicator is gone, you should see the profiler flow
- Answer the questions one by one. Validate that the `site_creation_step` is tracked as before.
- After you answer all questions, you should land on the loading screen or the dashboard screen if the site creation is complete.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/524475/6c9bb95d-c803-4855-ae70-9b8da01d40bc


---
